demisto_sdk/commands/common/tests/content_entity_validator_test.py

```diff
@@ -402,3 +405,153 @@ def test_is_backward_compatible(current_file, old_file, answer, error):
+            MODELING_RULE,
+            {"id": "ModelingRule", "name": "Modeling Rule"},
+            "",
+            True,
+        ),
+        (
+            "MyRuleModelingRules_1_!@#",
+            MODELING_RULE,
+            {"id": "ModelingRule", "name": "Modeling Rule"},
+            "\nThe file name should end with 'ModelingRules.yml'",
+            False,
+        ),
+        (
+            "MyRule",
+            PARSING_RULE,
+            {"id": "parsing-rule", "name": "Parsing-Rule"},
+            "\nThe file name should end with 'ParsingRules.yml'\nThe rule id should end with 'ParsingRule'\nThe rule name should end with 'Parsing Rule'",
+            False,
+        ),
+        (
+            "MyRule",
+            PARSING_RULE,
+            {"id": "ParsingRule", "name": "Parsing Rule"},
+            "\nThe file name should end with 'ParsingRules.yml'",
+            False,
+        ),
+        (
+            "MyRuleParsingRules",
+            PARSING_RULE,
+            {"id": "parsing-rule", "name": "Parsing Rule"},
+            "\nThe rule id should end with 'ParsingRule'",
+            False,
+        ),
+        (
+            "MyRuleParsingRules",
+            PARSING_RULE,
+            {"id": "ParsingRule", "name": "Parsing-Rule"},
+            "\nThe rule name should end with 'Parsing Rule'",
+            False,
+        ),
+        (
+            "MyRuleParsingRules",
+            PARSING_RULE,
+            {"id": "ParsingRule", "name": "Parsing Rule"},
+            "",
+            True,
+        ),
+        (
+            "MyRuleParsingRules_1_3",
+            PARSING_RULE,
+            {"id": "ParsingRule", "name": "Parsing Rule"},
+            "",
+            True,
+        ),
+        (
+            "MyRuleParsingRules_1_!@#",
+            PARSING_RULE,
+            {"id": "ParsingRule", "name": "Parsing Rule"},
+            "\nThe file name should end with 'ParsingRules.yml'",
+            False,
+        ),
+    ],
+)
+def test_is_valid_rule_suffix(
+    mocker, repo, rule_type, rule_file_name, rule_dict, expected_error, valid
+):
+    """
+    Given: A modeling/parsing rule with valid/invalid file_name/id/name
+        case 1: Wrong modeling rule file_name id and name.
+        case 2: Wrong modeling rule file_name.
+        case 3: Wrong modeling rule id.
+        case 4: Wrong modeling rule name.
+        case 5: Correct modeling rule file_name id and name.
+        case 6: Correct modeling rule file_name (with version) id and name.
+        case 7: Wrong modeling rule file_name (wrong version).
+        case 8: Wrong parsing rule file_name id and name.
+        case 9: Wrong parsing rule file_name.
+        case 10: Wrong parsing rule id.
+        case 11: Wrong parsing rule name.
+        case 12: Correct parsing rule file_name id and name.
+        case 13: Correct parsing rule file_name (with version) id and name.
+        case 14: Wrong parsing rule file_name (wrong version).
```

Comment:
please add these as comments to the test cases, so we can debug/fix them in the future without having to count manually